bundle/Resources/public/js/views/plugins/cof-copycontentplugin.js

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (C) eZ Systems AS. All rights reserved.
+ * For full copyright and license information view LICENSE file distributed with this source code.
+ */
+YUI.add('cof-copycontentplugin', function (Y) {
+    "use strict";
+    /**
+     * Provides the Copy Content plugin.
+     *
+     * @module cof-copycontentplugin
+     */
+    Y.namespace('cof.Plugin');
+
+    /**
+     * Provides plugin for the copy content plugin.
+     *
+     * @namespace cof.Plugin
+     * @class CopyContent
+     * @constructor
+     * @extends eZ.Plugin.CopyContent
+     */
+    Y.cof.Plugin.CopyContent = Y.Base.create('cofCopyContentPlugin', Y.eZ.Plugin.CopyContent, [], {
```

Comment:
why only handling copy ? I think the same problem will happen with any use of the UDW requiring the Content Type to be a container like moving or adding a Location or any custom or future use of the UDW with the same constraint. So most likely a very first thing would be to improve the UDW so that it carries some meta information.